macros/src/gtk.rs

```diff
@@ -276,14 +276,18 @@ pub fn expand_handler(
             vgtk::lib::glib::MainContext::ref_thread_default().spawn_local(
                 async move {
                     let msg = async move { #body_s }.await;
```

Comment:
There should be a `use vgtk::ext::IntoSignalReturn;` here, or else all callers of the `gtk!` macro that contain handlers need to import it.